src/engine/game/effects/recruitmessage.lua

```diff
@@ -95,7 +94,7 @@ function RecruitMessage:draw()
 
     if (self.second_number > 1) then
         love.graphics.setFont(Assets.getFont("goldnumbers"))
-        love.graphics.print(tostring(self.first_number), self.texture:getWidth() - 70, 35)
+        love.graphics.print(tostring(self.first_number), self.texture:getWidth() - 70 - ((#tostring(self.first_number) - 1) * 20), 35)
```

Comment:
This is inaccurate. In Deltarune, the code is as follows:
```gml
draw_self()
if (secondnumber > 1)
{
    draw_set_color(c_white)
    draw_set_font(global.damagefontgold)
    draw_set_alpha(image_alpha)
    draw_text((x - 70), (y + 35), firstnumber)
    draw_text((x - 30), (y + 35), secondnumber)
    draw_sprite(spr_numbersfontbig_gold, 13, (x - 50), (y + 35))
    draw_set_alpha(1)
}
```

Reply:
This is just an offset which fixes overlapping when the count is higher than 9. There're no enemies in Deltarune with that number.